binding.gyp

```diff
@@ -32,6 +32,7 @@
             }
           },
         }, {
+          'libraries': ['-lzmq'],
```

Comment:
I've actually no idea what this does.
Since we now have the same gyp file for windows as https://github.com/JustinTulloss/zeromq.node we can ensure the windows fallback works if no windows binaries are uploded.